assets/js/theme/common/carousel/utils/analizeSlides.js

```diff
@@ -1,4 +1,4 @@
-const FOCUSABLE_ELEMENTS_SELECTOR = '[href], button, input, textarea, select, details, [contenteditable="true"], [tabindex]';
+import { FOCUSABLE_ELEMENTS_SELECTOR } from './refreshFocus';
```

Comment:
Ideally, it should be moved into constants file

Reply:
I was thinking about that:) so should we create separate file for only one variable?

Reply:
I will approve, but this decision is up to you

Reply:
created separate file - law is law:)